### ISSUE TITLE
TMP installation instructions

### ISSUE THREAD

**Issue by faceleg** [caiogondim/maglev#14]
Do the current instructions work?

I use:

```
set -g @tpm_plugins '                \                                                                            
    tmux-plugins/tpm                 \                                                                            
    tmux-plugins/tmux-sensible       \                                                                            
    tmux-plugins/tmux-resurrect      \                                                                            
    tmux-plugins/tmux-continuum      \                                                                            
    tmux-plugins/tmux-yank           \                                                                            
    tmux-plugins/tmux-pain-control   \                                                                            
    tmux-plugins/tmux-copycat        \                                                                            
    tmux-plugins/tmux-open           \                                                                            
    tmux-plugins/tmux-battery        \                                                                            
    tmux-plugins/tmux-cpu            \                                                                            
    caiogondim/maglev                \
'
```

**Comment by caiogondim** [comment]
It was broken.
And with your instructions it is okay, now.

Thanks so much, @faceleg 

Fixed on ea67b36ea78a322efc8fb1fa7068e94f46634efb

**Comment by faceleg** [comment]
Yay!
